back-office/shop-parameters/customer-settings/groups.md

```diff
@@ -0,0 +1,141 @@
+
+**Discount (%).** Only numbers can be typed in the field. If the merchant tries to save letters or a negative value, then he should stay on this modal window and get an error notification right below the field: `The discount value is incorrect (must be a percentage).` in `Admin.Shopparameters.Notification`.
+
+Clicking 'Add' saves preferences. It displays the category discount(s) in the 'Edit' form, just below the 'Category discount' field, with the following information: route to the category, discount value, and a 'Delete' button. If a user deletes a discount, the line simply disappears.
+
+
+### Authorized modules
+
+With this setting, **merchants can authorize or prevent members of this group to access and use some of the shop's modules**. A list of all the store's enabled modules is displayed with a switch button to enable or disable each module. By default, all modules are enabled.
+
+Two actions are available at the end of the page: _Save_ and _Cancel_. When saving, the user gets redirected to the Shop Parameters > Customer Settings > Groups page with a success notification: 'Successful update.'.
+
+
+## View form
+
+A summary of the preferences set for a customers group is accessible through the 'View' action or just by clicking on the line in the groups listing. It opens a new page divided into three parts:
+
+### Group information
+
+Merchants can find all settings in this part:
+
+- Name of the group of customers<br>
+- Discount<br>
+- Price display method<br>
+- Show prices
+
+
+### Current category discount
+
+A warning notification, _None_, is displayed by default if the customers group has no category discount. If there is one or several, it takes the form of a listing displaying the following information:
+
+- Category<br>
+- Discount
+
+ 
+### Members of this customer group
+
+A wording is supposed to explicit how group members are displayed: `Limited to the first 100 customers.` and `Please use filters to narrow your search.` in `Admin.Shopparameters.Feature`.
+
+By default, the table is divided into the following columns:
+
+- ID<br>
+- Social title<br>
+- First names<br>
+- Last name<br>
+- Email address<br>
+- Date of birth<br>
+- Registration date<br>
+- Enabled
+
+Searching fields are available for users to narrow down their search. 
+
+At the beginning of each line is a checkbox while, at the end of each line, an 'Edit' button can be clicked, redirecting to the Customer dedicated page in the Customers > Customers section of the back office. Checkboxes play with the 'Bulk actions' selector that allows either to 'Select all', 'Unselect all', 'Enable selection', or 'Disable selection'.
```

Comment:
Humm, weird because I have the bulk actions but no checkboxes :thinking: 
Maybe a bug ?

![capture d'écran_2910](https://user-images.githubusercontent.com/13449658/102527387-50870080-409d-11eb-8894-c39bb1829ae5.png)

Reply:
Yes, https://github.com/PrestaShop/PrestaShop/issues/21894 :tada: